.travis.yml

```diff
@@ -9,7 +9,7 @@ matrix:
         - sudo apt-get update
         - sudo apt-get install -y --allow-unauthenticated libopenblas-dev liblapack-dev g++ libboost-all-dev python-pip cython python-numpy python-pandas
         - sudo pip install --upgrade --ignore-installed setuptools cython
-        - curl https://ftp.fau.de/macports/distfiles/armadillo/armadillo-6.500.5.tar.gz | tar xvz && cd armadillo*
+        - curl https://ftp.fau.de/macports/distfiles/armadillo/armadillo-8.400.0.tar.xz | tar xv && cd armadillo*
```

Comment:
In ensmallen I used:

``curl https://ftp.fau.de/macports/distfiles/armadillo/armadillo-8.400.0.tar.xz | tar -xvJ && cd armadillo*;``

and also installed ``xz-utils`` not sure that step is necessary.

Reply:
Yeah, I made the same change here... let's see if it works.  I don't mind installing `xz-utils` even if it's not needed. :)